list/fleet.cattle.io.bundle.vue

```diff
@@ -1,28 +1,72 @@
 <script>
+import { FLEET } from '@/config/types';
+import Banner from '@/components/Banner';
+import Loading from '@/components/Loading';
 import ResourceTable from '@/components/ResourceTable';
 import {
   AGE,
   STATE,
   NAME,
 } from '@/config/table-headers';
+import { isHarvesterCluster } from '@/utils/cluster';
 
 export default {
   name:       'ListBundle',
-  components: { ResourceTable },
+  components: {
+    Banner, Loading, ResourceTable
+  },
 
   props: {
     schema: {
       type:     Object,
       required: true,
     },
+  },
 
-    rows: {
-      type:     Array,
-      required: true,
-    },
+  async fetch() {
+    this.allBundles = await this.$store.dispatch('management/findAll', { type: FLEET.BUNDLE });
+    this.allFleet = await this.$store.dispatch('management/findAll', { type: FLEET.CLUSTER });
+  },
+
+  data() {
+    return {
+      allFleet:   [],
+      allBundles: [],
+    };
   },
 
   computed: {
+    harvesterClusters() {
+      const harvester = {};
+
+      this.allFleet.forEach((c) => {
+        if (isHarvesterCluster(c)) {
+          harvester[c.metadata.name] = c;
+        }
+      });
+
+      return harvester;
+    },
+
+    bundles() {
+      const harvester = this.harvesterClusters;
+
+      return this.allBundles.filter((bundle) => {
+        const targets = bundle.spec?.targets || [];
+
+        // Filter out any bundle that has one target whose cluster is a harvester cluster
+        if (targets.length === 1) {
+          return !harvester[targets[0].clusterName];
+        }
```

Comment:
It seems like we want to omit bundles where the only target/s are harvester.

Would it be possible for a bundle to have multiple targets and all of it's targets be harvester clusters? If so, would we want to omit those?

Reply:
Thanks Cody - I was guarding against the bundle that gets created by default in the Harvester cluster that we don't want to show - I'll merge as is and check about bundle creation with multiple Harvester clusters.